src/Headers/DomainBox.h

```diff
@@ -75,12 +75,12 @@ MPI_Datatype CreateBoxType (Box<ndim> dummy) {
 /// \date   03/04/2013
 //=================================================================================================
 template <int ndim>
-struct DomainBox : public Box<3> {
-  boundaryEnum boundary_lhs[3];     ///< LHS boundary types
-  boundaryEnum boundary_rhs[3];     ///< RHS boundary types
-  FLOAT size[3];                    ///< Side-lengths of bounding box
-  FLOAT half[3];                    ///< Half side-lengths of bounding box
-  bool PeriodicGravity ;               ///< Whether the domain is using periodic gravity.
```

Comment:
The comment said that there was a reason why everything was 3d (but I don't know how important it is). If the comment is not correct anymore for whatever reason, then it should be removed. If instead it is correct that here we hard-code 3d, then we might just as well remove the template from the class...